tembo-cli/src/cmd/login.rs

```diff
@@ -0,0 +1,137 @@
+use actix_cors::Cors;
+use actix_web::{http::header, post, web, App, HttpResponse, HttpServer, Responder};
+use anyhow::{Error, Result};
+use clap::Args;
+use serde::Deserialize;
+use std::fs;
+use std::io::{self, Write};
+use std::sync::Arc;
+use tokio::sync::Notify;
+use tokio::time::{self, Duration};
+use webbrowser;
+
+#[derive(Args)]
+pub struct LoginCommand {}
+
+#[derive(Deserialize)]
+struct TokenResponse {
```

Comment:
We should rename this to `TokenBody` or `TokenRequest` since this is a request body type and not really a response